src/generated/resources/assets/gtceu/lang/en_us.json

```diff
@@ -3350,6 +3352,14 @@
   "gtceu.machine.machine_hatch.locked": "Machine Interface Locked",
   "gtceu.machine.machine_hatch.processing_array": "When in the §eProcessing Array§7, only holds machines that work in the §eProcessing Array",
   "gtceu.machine.machine_hatch.tooltip": "Specialized Access Bus that only holds valid items",
+  "gtceu.machine.magic_energy_absorber.name": "Magic Energy Absorber",
+  "gtceu.machine.magic_energy_absorber.tooltip.0": "Max Voltage OUT: §a2048 §7(§5EV§7)",
+  "gtceu.machine.magic_energy_absorber.tooltip.1": "Collects energy from end crystals on naturally generated obsidian pillars in the End within 64 blocks.",
```

Comment:
Prefix all uncolored text in tooltips with §7

Reply:
![image](https://github.com/user-attachments/assets/08326433-f13e-426d-b195-81995b9a4e12)